Code_Exercises/Exercise_02_First_Kernel/solution.cpp

```diff
@@ -0,0 +1,49 @@
+/*
+ * SYCL Quick Reference
+ * ~~~~~~~~~~~~~~~~~~~~
+ *
+ * // Include SYCL header
+ * #include <CL/sycl.hpp>
+ *
+ * // Default construct a queue
+ * auto q = sycl::queue{};
+ *
+ * // Allocate device memory
+ * auto * devPtr = malloc_device<T>(mycount, q);
+ *
+ * // Memcpy
+ * auto event1 = q.memcpy(dst, src, sizeof(T)*n, {dep_event1, dep_event2});
+ *
+ * // Free memory
+ * free(ptr);
+ *
+ * // single task
+ * auto event2 = q.single_task(global_range, {event1, event2}, lambda);
```

Comment:
Worth giving a couple of examples of single task here? e.g also have 
defaultQueue.single_task([=] { dev_a[0] = dev_a[0] * 2; });

Reply:
Yes good idea.